src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -31,8 +31,11 @@ class ItemRecognition extends Component {
     modelLoaded: false
   };
 
+  predictionStack = [];
+
   componentDidMount() {
     this.props.setPrediction(null, null);
+    this.predictionStack = [];
```

Comment:
Is this needed in `componentDidMount` as well?

Reply:
Wanted to make sure it was empty at the start of each use. Only defined it outside of componentDidMount to make sure it existed the first time.